packages/components/src/components/user/UserDetailsPanel.spec.tsx

```diff
@@ -35,12 +35,50 @@ describe("<UserDetailsPanel/>", () => {
         });
     });
 
-    test("with principal", (done) => {
+    test("with principal no buttons because of self", (done) => {
         const component = (
             <UserDetailsPanel
                 userId={1004}
```

Comment:
maybe make the test user ids constants to make reading the intents a little clearer

Reply:
Good idea. Done.